fluss-server/src/test/java/com/alibaba/fluss/server/log/LogManagerTest.java

```diff
@@ -209,6 +210,37 @@ void testRecoveryAfterLogManagerShutdown(String partitionName) throws Exception
         newLogManager.shutdown();
     }
 
+    @ParameterizedTest
+    @MethodSource("partitionProvider")
+    void testHasCleanShutdownMarkerAfterLogManagerShutdown(String partitionName) throws Exception {
```

Comment:
nit: I think we don't need to parameterized this test, as the partition doesn't affect the logic of shutdown flag.

Reply:
Done!